apps/backend/app/api/routes/user_data.py

```
@@ -0,0 +1,353 @@
+            s3_key = s3_url.split("/")[-1]
+
+        print(f"Extracted S3 key: {s3_key}")
+        print(f"Attempting to get S3 object with key: {s3_key}")
+
+        # Get the file from S3
+        try:
+            response = s3_client.get_object(
+                Bucket=os.getenv("AWS_BUCKET_NAME"),
+                Key=s3_key,
+            )
+        except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)